src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java

```diff
@@ -0,0 +1,347 @@
+            if (! inited) {
+                dict.put("CFBundleIdentifier", bundleIdName);
+                dict.put("CFBundleExecutable", executableName);
+                dict.put("CFBundleName", projectConfiguration.getAppName());
+                dict.saveAsXML(plist);
+            }
+            dict.put("DTPlatformName", xcodeUtil.getPlatformName());
+            dict.put("DTSDKName", xcodeUtil.getSDKName());
+            dict.put("MinimumOSVersion", "11.0");
+            dict.put("CFBundleSupportedPlatforms", new NSArray(new NSString("iPhoneOS")));
+            dict.put("DTPlatformVersion", xcodeUtil.getPlatformVersion());
+            dict.put("DTPlatformBuild", xcodeUtil.getPlatformBuild());
+            dict.put("DTSDKBuild", xcodeUtil.getPlatformBuild());
+            dict.put("DTXcode", xcodeUtil.getDTXCode());
+            dict.put("DTXcodeBuild", xcodeUtil.getDTXCodeBuild());
+            NSDictionaryEx orderedDict = new NSDictionaryEx();
+            orderedDict.put("CFBundleVersion", dict.get("CFBundleVersion"));
+            dict.remove("CFBundleVersion");
+            dict.getKeySet().forEach(k -> orderedDict.put(k, dict.get(k)));
+
+            if (partialPListDir != null) {
+                Files.walk(partialPListDir)
+                        .filter(f -> f.toString().endsWith(".plist"))
+                        .forEach(f -> {
+                            try {
+                                NSDictionary d = (NSDictionary) PropertyListParser.parse(f.toFile());
+                                d.keySet().forEach(k -> orderedDict.put(k, d.get(k)));
+                            } catch (Exception e) {
+                                Logger.logSevere(e, "Error reading plist");
+                            }
+                        });
+            }
+            orderedDict.put("MinimumOSVersion", minOSVersion != null ? minOSVersion : "11.0");
+
+            //             BinaryPropertyListWriter.write(new File(appDir, "Info.plist"), orderedDict);
+            orderedDict.saveAsBinary(workDir.resolve("Info.plist"));
+            orderedDict.saveAsXML(tmpPath.resolve("Info.plist"));
+            orderedDict.getEntrySet().forEach(e -> {
+                        if (e.getKey().equals("CFBundleIdentifier")) {
+                            Logger.logDebug("Bundle ID = "+e.getValue().toString());
+                            bundleId = e.getValue().toString();
+                        }
+                        Logger.logDebug("Info.plist Entry: " + e);
+                    }
+            );
+            return plist;
+        } catch (Exception ex) {
+            Logger.logSevere(ex, "Could not process property list");
+        }
+        return null;
+    }
+
+    static Path getPlistPath(ProcessPaths paths, String sourceName) {
+        Path userPlist = paths.getSourcePath().resolve(sourceName).resolve(Constants.PLIST_FILE);
+        if (userPlist.toFile().exists()) {
+            return userPlist;
```

Comment:
`Files.exists(userPlist)` is the new API

Reply:
done